docs/reduction-in-locks.md

```diff
@@ -0,0 +1,98 @@
+# Reduced backup lock time
+
+!!! important
+
+    The `--lock-ddl=REDUCED` option is a [tech preview](./glossary.md#tech-preview). Before using this option in production, we recommend that you test restoring production from physical backups in your environment, and also use the alternative backup method for redundancy.
+
+Percona Server for MySQL implemented `LOCK TABLES FOR BACKUP` to block Data Definition Language (DDL) statements on a server. Percona XtraBackup utilizes this lock during the backup process to ensure that any `DDL` event does not corrupt the backup. The lock does not affect the Data Manipulation Language (DML) statements.
+
+## How the backup lock works
+
+With the [`--lock-ddl=ON`](./xtrabackup-option-reference.md/#lock-ddl) option the backup lock is enabled by default from the start of the backup process.
+
+* Set a backup lock on the server to prevent any `DDL` operations, ensuring no new tables are created, renamed, or dropped during the backup.
+
+* Copy all redo logs from the checkpoint up to the current Log Sequence Number (LSN) and start tracking new entries.
```

Comment:
Copy all redo logs - the "all" is redundant

Reply:
Tracking is done as part of  copy redo log step. Actually, copy redo log is done in two steps
1. parse redo log (we now track the file operations here)
2. copy redo log